Tools/electrostatic_sphere_plot.py

```diff
@@ -0,0 +1,149 @@
+import yt
```

Comment:
Is `electrostatic_sphere_plot.py` a commonly used file or should it better be places in the same directory as the input file?